sycl/test/regression/macro_conflict.cpp

```diff
@@ -0,0 +1,25 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+// expected-no-diagnostics
```

Comment:
This line is no-op if `-verify` option wasn't specifeid

Reply:
okay I will add -verify as the compilation option.